### ISSUE TITLE
Luminosity ratio is less than 4.5:1 for some Text (Home, Setting, More) showing on menu bar

### ISSUE THREAD

**Issue by Helen09** [Microsoft/pxt-chibitronics#203]
**Repro steps:**
• Navigate to https://makecode.chibitronics.com/#editor
• Navigate to the Home control lying in the header section on the page and select it.
• Verify the luminosity ratio for all the Text content.

**Expected result:**
Luminosity ratio should be more than 4.5:1 for some Text(Home, Setting, More) showing on menu bar
**Actual result:**
Luminosity ratio is less than 4.5:1 for some Text (Home, Setting, More) showing on menu bar
![issue1](https://user-images.githubusercontent.com/23466737/39855729-95b71b92-541c-11e8-97e5-bc683abbda8d.png)

**Comment by samelhusseini** [comment]
Yeah.. that's Chibi's colors.. I'll try white and see what I can do.
